src/main/java/uk/gov/hmcts/reform/bulkscan.payment.processor/Application.java

```diff
@@ -2,10 +2,13 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.config.SiteConfiguration;
 
 @SpringBootApplication
 @EnableCircuitBreaker
+@EnableConfigurationProperties(SiteConfiguration.class)
```

Comment:
It is normally recommended to use this on the component where properties ought to be injected

Reply:
there is no component yet, I added to see config works in pipeline.

Reply:
ah, cool beans. add a `todo` maybe at the end? so it's clear this will go away once component comes to life

Reply:
config worked fine in pipeline.
removed the annotation .